content/zh-tw/devops.md

```diff
@@ -0,0 +1,21 @@
+---
+title: DevOps
+status: Completed
+category: 概念
+tags: ["方法論", "", ""]
+---
+
+## 什麼是 DevOps {#what-it-is}
+
+DevOps 代表著由團隊主導應用程式開發（development）到正式維運（operations）完整過程的一種方法，所以它才被稱作 DevOps。DevOps 不只是一套技術的運用，更是文化與流程的全面轉變。DevOps 要求工程團隊專注在更小的元件，而非一次處理整個功能，並且減少交接的次數，因為頻繁的交接正是導致錯誤的常見原因。
```

Comment:
```suggestion
DevOps 代表著由團隊主導應用程式開發到正式維運完整過程的一種方法，所以它才被稱作 DevOps。DevOps 不只是一套技術的運用，更是文化與流程的全面轉變。DevOps 要求工程團隊專注在更小的元件，而非一次處理整個功能，並且減少交接的次數，因為頻繁的交接正是導致錯誤的常見原因。
```

There is no need to explicitly indicate the English lang.

Reply:
Because "DevOps" is a portmanteau of "Development" and "Operations", some rookies might not know the source of "DevOps" if removing the original words. Besides, I also refer to ko team (https://glossary.cncf.io/ko/devops/), which also keep "development" and "operations" in their localization.

Or, maybe... better to be translated into?

1. 應用程式開發（Dev）到正式維運（Ops）
2. 應用程式開發（development, "Dev"）到正式維運（operations, "Ops"）